index.js

```diff
@@ -6,9 +6,12 @@ var supportsColor = require('supports-color');
 var defineProps = Object.defineProperties;
 var isSimpleWindowsTerm = process.platform === 'win32' && !/^xterm/i.test(process.env.TERM);
 
+var levelMapping = ['ansi', 'ansi', 'ansi256', 'ansi16m'];
+var skipModels = ['gray'];
```

Comment:
No - "model" refers to the color models within color-convert. `gray` was for CSS `gray()` and conflicts with our color `gray` or `grey`.